src/components/Input/Input.tsx

```diff
@@ -132,7 +132,7 @@ function Input(props: Props) {
             disabled={disabled}
             ref={inputRef}
           />
-          {(endIcons?.length ?? isOverlay) && (
+          {((endIcons?.length ?? 0) > 0 || isOverlay) && (
```

Comment:
Does it not mean the same thing?

Reply:
Yeah, that was changed to make myself clear when debugging but forgot to change back